src/lib/plugins/sast/index.ts

```diff
@@ -78,13 +79,7 @@ export const codePlugin: EcosystemPlugin = {
     } catch (error) {
       let err: Error;
       if (isCodeClientError(error)) {
-        const isUnauthorized = isUnauthorizedError(error)
-          ? 'Unauthorized: '
-          : '';
-        err = new FailedToRunTestError(
-          `${isUnauthorized}Failed to run 'code test'`,
-          error.statusCode,
-        );
+        err = throwCodeClientError(error);
```

Comment:
nit. This function does not really throw an error, the name could be improved (I know, the worst type of comments 😸)

```suggestion
        err = resolveCodeClientError(error);
```